tox.ini

```diff
@@ -1,52 +1,50 @@
 [tox]
-envlist = py{37,36}-offline,py{37,36}-{online,astropydev,numpydev,build_docs},figure,doc_tests
+envlist = py{36,37}-{offline,online,astropydev,numpydev,build_docs},figures,conda
 
 [testenv]
-install_command = pip install -U {opts} {packages}
 setenv =
     MPLBACKEND = agg
     COLUMNS = 180
 deps =
-    astropydev: git+git://github.com/astropy/astropy
-    numpydev: git+git://github.com/numpy/numpy
-    build_docs: sunpy-sphinx-theme
-    build_docs: git+git://github.com/hawkowl/towncrier
-    codecov
-    offline,online,build_docs,astropydev: asdf
-    pytest-sunpy
```

Comment:
I ended up deleting pytest-sunpy from PyPi. In hindsight, I don't think we need it.